rfc/rfc-0019-rfc-process.md

```diff
@@ -0,0 +1,112 @@
+- The FCP will last 7 days. If there's unanimous agreement among the team the FCP can close early.
+- For voting, the binding votes are comprised of the maintainer team. Acceptance requires a super majority of binding votes in favor. The voting options are the following: Affirmative, Negative, and Abstinence. Non-binding votes are of course welcome. Super majority means 2/3 or greater.
+- If no substantial new arguments or ideas are raised, the FCP will follow the outcome decided. If there are substantial new arguments, then the RFC will go back into development.
+
+Once an RFC has been accepted, the team member who merges the pull request should do the following:
+
+- Assign an id based off the latest available number.
+- Rename the file based off the id inside `rfc/`.
+- Fill in the remaining metadata at the top.
+- Commit everything.
+
+### Status Field
+The status field in the meta section has the following acceptable values:
+- **Draft**: The maintainer team has not yet approved the RFC. Edits are still being made.
+- **Approved**: The maintainer team has approved the RFC and is accepting PRs for its implementation.
+- **On Hold**: The maintainer team has approved the RFC, but PRs are currently not being accepted for its implementation. A reason must be provided in the RFC.
+- **Superseded**: The maintainer team has approved the RFC, but it is replaced by another RFC.
+
+RFCs that are rejected will no longer be pulled into the `rfc/` directory.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Though RFCs are intended to be lightweight, this introduces extra process than what we're doing today. The additional work may make consistency hard.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- Stick with the current process, which is not well-defined, does not include a formal voting on approval of RFCs, and does not clearly indicate whether RFCs are in draft, accepted, rejected, or implemented. This makes it hard for new people to know how to propose changes, and how things are decided. It also makes it hard to track major changes to the project.
+
+# Prior Art
+[prior-art]: #prior-art
+
+The basic format of RFCs was [invented in the 1960s by Steve Crocker](https://en.wikipedia.org/wiki/Request_for_Comments#History). The current RFC process has been derived heavily from [Buildpacks](https://github.com/buildpacks/rfcs/). Many other open source projects have adopted an RFC process:
+
+- [Rust](https://github.com/rust-lang/rfcs)
+    - FCP lasts 10 days
+- [Ember](https://github.com/emberjs/rfcs)
+    - FCP lasts 7 days
+- [Yarn](https://github.com/yarnpkg/rfcs)
+    - FCP lasts 7 days
+- [React](https://github.com/reactjs/rfcs)
+    - FCP lasts 3 days
+- [IETF](https://www.rfc-editor.org/rfc/rfc2026.txt)
+
+# Unresolved Questions
+[unresolved-questions]: #unresolved-questions
+
+- What clearly defines a "substantial" change?
+- How long should the Final Comment Period be?
```

Comment:
I think you've answered these questions in the RFC, I'm happy with those answers.